core/state/statedb_test.go

```diff
@@ -727,3 +741,142 @@ func TestMissingTrieNodes(t *testing.T) {
 		t.Fatalf("expected error, got root :%x", root)
 	}
 }
+
+func addr(a string) common.Address {
+	return common.HexToAddress(a)
+}
+func slot(a string) common.Hash {
+	return common.HexToHash(a)
+}
```

Comment:
Nit. If you only use these methods in a single test, could you declare them as variables inside? Just trying to keep the statedb namespace a bit cleaner.